pgjdbc/src/test/java/org/postgresql/test/jdbc3/StringTypeParameterTest.java

```diff
@@ -17,15 +22,11 @@
 import java.sql.Types;
 import java.util.Properties;
 
-public class StringTypeParameterTest extends TestCase {
+public class StringTypeParameterTest {
 
   private Connection _conn;
 
-  public StringTypeParameterTest(String name) {
-    super(name);
-  }
-
-  protected boolean setUp(String stringType) throws Exception {
+  private boolean prepare(String stringType) throws Exception {
```

Comment:
I think it's worth to remove checks `TestUtil.isProtocolVersion(_conn, 3)` because this driver is "compatible with PostgreSQL 8.2 and higher using the version 3.0 of the protocol". Therefore this check always returns `true`